docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
 redelegations.
 
 <a name="q9"></a>
 #### 9. Zonemaster and privacy
 Since Zonemaster is open to everyone it is possible for anyone to check your
-domain and also see previous tests, however there is no way to tell
-who has run a specific test since nothing is logged except the time of the test.
+domain and its tests history.
+However there is no way to tell who has run a specific test since nothing is
+logged except the start time of the test.
 
 <a name="q10"></a>
-#### 10. How come I cannot test my domain?
-If we disregard the situation where the domain does not exist, as in when you input a
-non-existing domain to Zonemaster, there are 2 other possibilites:
-  - To protect the engine from multiple identical inputs, that is the same IP
-    checking the same zone several times, there is a delay of 5 minutes between
-    identical subsequent tests. This means that you can only test the
-    same domain once every 5 minutes, if you try and test it again within 5 minutes
-    the last results will be displayed instead.
-  - Because Zonemaster was made to check domains or DNS zones, like zonemaster.net, and
-    not hostnames in a domain (zone), like www.zonemaster.net, the Zonemaster will
-    report a failure if you try to test a host name instead of a domain matching a
+#### 10. How come my domain cannot be tested?
+If we disregard the situation where a domain does not exist, such as when a
+non-existing domain is input to Zonemaster, there are 2 other possibilities:
+  - To protect the Engine from repetitive inputs, being the same IP
+    checking the same domain several times, there is a delay of 5 minutes between
+    subsequently identical tests.
+    This means that you can only test the same domain once every 5 minutes,
+    and if you try and test it again during that time the last results
+    will be displayed instead.
+  - Because Zonemaster was made to check domains, or DNS zones (e.g. 'zonemaster.net'), and
+    not hostnames in a domain (e.g. 'www.zonemaster.net'), Zonemaster will
+    report a failure if you try to test a hostname instead of a domain matching a
     DNS zone.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain tested and
+also to the name servers hosting the parent zone of the domain tested.
 
-To get a full
-view of what queries and results are generated you can run the
-CLI interface (and in order to run the CLI interface you need to download the
-complete package and install it) and select full output.
-The output from the CLI tool is quite heavily technical
-so unless you are into bits and bytes you might want to skip this step.
+For the time being, the GUI interface of Zonemaster does not show any queries sent.
```

Comment:
It is not on the road map to show all details in the GUI interface. Remove "For the time being".

Reply:
Updated.